cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -51,6 +55,31 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
       );
     });
 
+    it('if a legacy MAP it should render a link using an a tag and href, rather than a plain text <link>', () => {
+      const requestPath = config[service].pageTypes[pageType].path;
```

Comment:
`path` can be an array - there's an environment variable that is a more stable way to determine the url :)

Reply:
`Cypress.env('currentPath')`?

Reply:
aye that's it - sorry, had to go to a meeting to eat some chocolate oranges before I could find it 👍

Reply:
Cheers, updated